templates/bundler_audit.rake

```diff
@@ -4,9 +4,25 @@ if Rails.env.development? || Rails.env.test?
   namespace :bundler do
     desc "Updates the ruby-advisory-db and runs audit"
     task :audit do
+      prepare_bundler_audit_to_exit_with_error_if_update_fails
       %w(update check).each do |command|
         Bundler::Audit::CLI.start [command]
       end
     end
   end
+
+  def prepare_bundler_audit_to_exit_with_error_if_update_fails
+    Bundler::Audit::CLI.class_eval do
+      def update
+        say "Updating ruby-advisory-db ..."
+        if Bundler::Audit::Database.update!
```

Comment:
Dropping the prefix sadly fails with `NameError: uninitialized constant Database`.